<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskItemWindow</class>
 <widget class="QWidget" name="TaskItemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: white;
  width:30px;
  height:30px;
  background-color:transparent;
  border-width: 0px;
  border-radius: 3px;
   image: url(:/Images/images/icon_05.png);
}
QPushButton:hover
{
  background-color: #A9A9A9;
  
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: white;
  width:30px;
  height:30px;
  background-color:transparent;
  border-width: 0px;
  border-radius: 3px;
   image: url(:/Images/images/icon_06.png);
}
QPushButton:hover
{
  background-color: #A9A9A9;
  
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblProjectName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string>My Project</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widgetBackground" native="true">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>0</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
 background-color:white;
 border:1px solid #AAAAAA; 
 border-radius: 10px;
}
</string>
   </property>
   <widget class="QLabel" name="lblWorkers">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-width:0px;
}</string>
    </property>
    <property name="text">
     <string>1 worker</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblActive">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(57, 150, 7);
border-width:0px;</string>
    </property>
    <property name="text">
     <string>Active</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>widgetBackground</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>lblProjectName</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
